.github/workflows/build.yml

```diff
@@ -37,7 +39,7 @@ jobs:
     steps:
     # Pinned 1.0.0 version
     - uses: marocchino/action-workflow_run-status@54b6e87d6cb552fc5f36dbe9a722a6048725917a
```

Comment:
I think you will need to skip this step as well when event is not workflow run otherwise workflow will fail.

Reply:
yes i have the below condition

Reply:
sg